packages/notifi-dapp-example/src/components/VerifyBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import { CardView } from '@/app/notifi/dashboard/page';
+import { Icon } from '@/assets/Icon';
+import { FormField } from '@notifi-network/notifi-react-card';
+import React, { useMemo } from 'react';
+
+export type VerifyBannerProps = Readonly<{
+  unVerifiedDestinations: ReadonlyArray<FormField>;
+  setCardView: (cardView: CardView) => void;
+}>;
+
+export const VerifyBanner: React.FC<VerifyBannerProps> = ({
+  unVerifiedDestinations,
+  setCardView,
+}: VerifyBannerProps) => {
+  const unVerifiedDestinationsString = useMemo(() => {
```

Comment:
If we are going to have `useTarget` hook, we might want to also move this one to `useTarget` hook and rename it accordingly (maybe `unVerifiedTargets`?)

Reply:
will do in next pr with dashboard destination section